### ISSUE TITLE
Qdrant Index — Attribute Error on Query

### ISSUE THREAD

**Issue by sandkoan** [jerryjliu/gpt_index#376]
When running the qdrant [example notebook](https://github.com/jerryjliu/gpt_index/blob/main/examples/vector_indices/QdrantIndexDemo.ipynb), I have no issues until I actually query the index, at which point I have this error:

```
AttributeError                            Traceback (most recent call last)
Cell In[6], line 1
----> 1 response = index.query('What does the author think about lawyers?')

File ~/Code/benipers/env/lib/python3.10/site-packages/gpt_index/indices/base.py:399, in BaseGPTIndex.query(self, query_str, verbose, mode, **query_kwargs)
    384 query_config = QueryConfig(
    385     index_struct_type=self._index_struct.get_type(),
    386     query_mode=mode_enum,
    387     query_kwargs=query_kwargs,
    388 )
    389 query_runner = QueryRunner(
    390     self._llm_predictor,
    391     self._prompt_helper,
   (...)
    397     recursive=False,
    398 )
--> 399 return query_runner.query(query_str, self._index_struct)

File ~/Code/benipers/env/lib/python3.10/site-packages/gpt_index/indices/query/query_runner.py:112, in QueryRunner.query(self, query_str, index_struct)
    104 query_kwargs = self._get_query_kwargs(config)
    105 query_obj = query_cls(
    106     index_struct,
    107     **query_kwargs,
    108     query_runner=query_runner,
...
     89     )
     90     nodes.append(node)
     92     if similarity_tracker is not None:

AttributeError: 'ScoredPoint' object has no attribute 'get'
```

**Comment by kacperlukawski** [comment]
@sandkoan Thanks for pointing this out! It seems there was a bug introduced while merging Qdrant, but I'll provide a hotfix soon!

**Comment by kacperlukawski** [comment]
@sandkoan @jerryjliu I've provided a fix for that in #394

**Comment by sandkoan** [comment]
Many thanks!
